Modules/Segmentation/ConnectedComponents/include/itkConnectedComponentImageFilter.h

```diff
@@ -32,8 +32,11 @@ namespace itk
  * Each distinct object is assigned a unique label. The filter experiments
  * with some improvements to the existing implementation, and is based on
  * run length encoding along raster lines.
- * The final object labels start with 1 and are consecutive. Objects
- * that are reached earlier by a raster order scan have a lower
+ * If the output background value is set to zero (the default), the final 
+ * object labels start with 1 and are consecutive. If the output background 
+ * is set to a non-zero value (by calling the SetBackgroundValue() routine of the filter), 
+ * the final labels start at 0, and remain consecutive except for skipping the background 
+ * value as needed. Objects that are reached earlier by a raster order scan have a lower
```

Comment:
I am not sure that labeling skips the background value. Did you test that? I think that labeling will reuse background value if it is low enough (number of labels is grater than background value).

Reply:
And if so, that is a bug which needs fixing.